action.yaml

```diff
@@ -8,8 +8,8 @@ inputs:
     description: 'CodeDeploy Deployment Group Name'
     required: true
   appspec-file:
-    description: 'Appspec file'
+    description: 'AppSpec file'
     required: true
 runs:
-  using: 'node16'
+  using: 'node20'
```

Comment:
This is technically a breaking change; if anyone's using older GitHub Actions runner that doesn't support `node20`, they would go 💥.

Based on the way we're baking in BYU URLs into this action, I doubt we have any external consumers. And internally, everyone should be on a newer runner. Thus, I'm inclined to treat this as a minor version update.